openfecli/commands/gather.py

```diff
@@ -87,6 +87,9 @@ def dp2(v: float) -> str:
         result = load_results(result_fn)
         if result is None:
             continue
+        elif result['estimate'] is None or result['uncertainty'] is None:
+            print("WARNING: Calculations for "+result_fn+" did not finish succesfully!")
```

Comment:
```suggestion
            print(f"WARNING: Calculations for {result_fn} did not finish succesfully!")
```
I agree with @dwhswenson but if we do it this way, I'd prefer an f-string.